electrical/RPi BreakBeam.py

```diff
@@ -0,0 +1,24 @@
+import RPi.GPIO as GPIO
+
+BEAM_PIN = 17
+beam_broken = False
+
+def break_beam_callback(channel):
+    if GPIO.input(BEAM_PIN):
+        beam_broken = False
+        print(beam_broken)
+    else:
+        beam_broken = True
+        print(beam_broken)
+    return beam_broken

```

Comment:
This can be condensed into a few lines! 
if not GPIO.input(BEAM_PIN):
         beam_broken = True
print(beam_broken)
return beam_broken

Reply:
I agree!

Reply:
another cool simplification: 

`beam_broken = not GPIO.input(BEAM_PIN)
print(beam_broken)
return beam_broken`